ail_vru_uc1_scenario/cdasim_config/scenario-runner/vru_collision.xml

```diff
@@ -17,13 +17,13 @@ limitations under the License.
 -->
 
 <scenarios>
-    <scenario name="VulnerableRoadUserCollision_LeftTurn" type="VulnerableRoadUserCollision" town="Town04">
+    <scenario name="VulnerableRoadUserCollision_North" type="VulnerableRoadUserCollision" town="Town04">
         <other_actor x="250.38" y="-179.00" z="1.0" model="walker.pedestrian.0002" rolename="crossing_person"/>
         <other_actor x="239.82" y="-173.39" z="0.2" yaw="180.0" model="vehicle.carlamotors.carlacola" rolename="heavy_vehicle_1"/>
         <other_actor x="245.82" y="-173.39" z="0.2" yaw="180.0" model="vehicle.carlamotors.carlacola" rolename="heavy_vehicle_2"/>
         <other_actor x="251.82" y="-175.00" z="0.2" yaw="145.0" model="vehicle.carlamotors.carlacola" rolename="heavy_vehicle_3"/>
     </scenario>
-    <scenario name="VulnerableRoadUserCollision_RightTurn" type="VulnerableRoadUserCollision" town="Town04">
+    <scenario name="VulnerableRoadUserCollision_South" type="VulnerableRoadUserCollision" town="Town04">
```

Comment:
Is there a particular reason why we are changing `LeftTurn` to `North` and `RightTurn` to `South`?

Reply:
Can these names match the name of the route or would it make sense to have independent environment variables for route and scenario.

Reply:
Because it is no longer "rightTurn". It is through movement. On the map the collision is happening on Northbound or Southbound road. Please see the images:
![image](https://github.com/usdot-fhwa-stol/carma-config/assets/20613282/b5e25634-6d51-4aca-a65b-e25b292a0fa0)
![image](https://github.com/usdot-fhwa-stol/carma-config/assets/20613282/a4180b1e-4d4e-43d8-ac0f-7a500729cbc9)

Clarification: South is not being used for the VRU use case. It is just there because we created it before. Feel free to delete it as well as the south route if you want

Reply:
That makes sense. Although we should keep the South scenario even if its not used for verification testing. It could be a useful regression test scenario in the future.

Reply:
Let me know if the new naming makes sense